packages/atoms/package.json

```diff
@@ -16,7 +16,7 @@
     "@vitejs/plugin-react": "^2.2.0",
     "rollup-plugin-node-builtins": "^2.1.2",
     "typescript": "^4.9.3",
-    "vite": "^3.2.4"
+    "vite": "^4.1.2"
```

Comment:
<img width="881" alt="Screenshot 2023-06-09 at 6 17 48 PM" src="https://github.com/calcom/cal.com/assets/1780212/ef81f351-cb6f-4f58-a55e-2f627181b63c">
@JeroenReumkens Couldn't test atoms build. I get this error locally which I am sure is unrelated.